pkgs/development/python-modules/gtimelog/default.nix

```diff
@@ -1,43 +1,58 @@
-{ stdenv
-, buildPythonPackage
-, pkgs
-, python
-, pygobject3
+{ stdenv, fetchFromGitHub, makeWrapper
+, glibcLocales, gobject-introspection, gtk3, libsoup, libsecret
+, buildPythonPackage, python
+, pygobject3, freezegun, mock
 }:
 
-buildPythonPackage rec {
+buildPythonPackage {
   pname = "gtimelog";
-  version = "0.9.1";
+  version = "unstable-2019-08-06";
 
-  src = pkgs.fetchurl {
-    url = "https://github.com/gtimelog/gtimelog/archive/${version}.tar.gz";
-    sha256 = "0qk8fv8cszzqpdi3wl9vvkym1jil502ycn6sic4jrxckw5s9jsfj";
+  src = fetchFromGitHub {
+    owner = "gtimelog";
+    repo = "gtimelog";
+    rev = "160fd4d5debad2d3d357d1ac1557f87e5dcb2bd4";
+    sha256 = "0lawbq7x03g6l0r8sn1lvf6n4yd90ywh6djhxvsa40na387lpvy9";
   };
 
-  buildInputs = [ pkgs.glibcLocales ];
+  buildInputs = [
+    makeWrapper
+    glibcLocales gobject-introspection gtk3 libsoup libsecret
+  ];
 
-  LC_ALL="en_US.UTF-8";
-
-  # TODO: AppIndicator
-  propagatedBuildInputs = [ pkgs.gobject-introspection pygobject3 pkgs.makeWrapper pkgs.gtk3 ];
+  propagatedBuildInputs = [
+    pygobject3 freezegun mock
+  ];
 
   checkPhase = ''
-    substituteInPlace runtests --replace "/usr/bin/env python" "${python}/bin/${python.executable}"
+    substituteInPlace runtests --replace "/usr/bin/env python3" "${python}/bin/python"
```

Comment:
${python.interpreter}

Reply:
Thanks for your feedback. I have updated the PR.